docs/release_notes.md

```diff
@@ -58,7 +58,7 @@ distribution. The new script provides additional flexibility in how it modifies
 the NCN personalization CFS configuration for SAT. It can modify an existing CFS
 configuration by name, a CFS configuration being built in a JSON file, or an
 existing CFS configuration that applies to certain components. For more information,
-see [Perform NCN Personalization](install.md#perform-ncn-personalization).
+see Perform NCN Personalization.
```

Comment:
This is kind of odd to refer to a section that does not exist anymore. Perhaps just get rid of the "For more information..." sentence entirely.

Reply:
Sounds good, I wasn't sure how you guys have handled situations like this in the past. I also prefer removing the sentence. Updating in next commit.

Reply:
This might be the first time we've hit the issue where we removed a section that's referenced from an older version's release notes.